deployment/helm-chart/values.yaml

```
@@ -0,0 +1,369 @@
+    engineBootstrap: "0.1.5"
+    aiService: "0.22.4"
+    ui: "0.27.4"
+    ibisServer: "0.15.13"
+    product: "0.22.2"
+
+secrets:
+  # Option 1: Use existing secret (recommended for production)
+  useExistingSecret: true
+  annotations: {}
+  existingSecretName: "wren-secret"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)